packages/openid4vc/src/openid4vc-issuer/router/metadataEndpoint.ts

```diff
@@ -11,6 +12,7 @@ export function configureIssuerMetadataEndpoint(router: Router) {
     (_request: OpenId4VcIssuanceRequest, response: Response, next) => {
       const { agentContext, issuer } = getRequestContext(_request)
 
+      const proofTypesSupported = getProofTypesSupported(agentContext)
```

Comment:
One thing that is unfortunate about this approach is that it means that an issuer now automatically indicates it supports all `alg_values` Credo support. Before you had to manually defines which alg values you want to support for the cryptosuites , etc.. 

Would it be possible to derive the proofTypesSupported from the cryptosuites supported alg values if it is not defined on the v11 metadata? 

Also I see you're transforming back to v11 format, but then you're also adding properties to the v11 format that are not v11. Would'nt it make sense in this case to just store it as v13 and then when you fetch it and it's v13, you transform it to v11, and if you fetch it and it's v11 you transform it to v13. That feels less hacky to me than adding v13 properties to the v11 format. Better to just have two different metadata storage formats IMO. We can even add an `issuerMetadataVersion` to the record (default value is `v1.draft11` because all records that don't have this property will imply draft 11) and we can also set it to `v1.draft13`. I understand you want to keep it consistent with v11 for now, but this feels hacky in my opnionion and loses that simplicity. So I'd prefer to also allow storing in v13 format